patches/0002-Add-S3-legacy-bucket-to-resources.patch

```diff
@@ -80,10 +80,10 @@ index 0000000000..fd117b2971
 +
 +func ResourceBucketLegacy() *schema.Resource {
 +	return &schema.Resource{
-+		Create: resourceBucketLegacyCreate,
-+		Read:   resourceBucketLegacyRead,
-+		Update: resourceBucketLegacyUpdate,
-+		Delete: resourceBucketLegacyDelete,
++		CreateContext: resourceBucketLegacyCreate,
```

Comment:
Was this fairly mechanical or needs careful review?

Reply:
Mostly mechanical. `AWSClient.Region` and  `AWSClient.Partition` are methods now